api/application/client.go

```diff
@@ -1020,3 +1020,24 @@ func (c *Client) MergeBindings(req params.ApplicationMergeBindingsArgs) error {
 	}
 	return results.OneError()
 }
+
+// UnitsInfo retrieves units information.
+func (c *Client) UnitsInfo(units []names.UnitTag) ([]params.UnitInfoResult, error) {
```

Comment:
I don't like how we return wire protocol structures from the api calls.

Perhaps this would be a good time to make a good example?

Reply:
I don't like it either. I was sticking to convention for this facade but will change this method